backtrace-library/src/main/java/backtraceio/library/metrics/BacktraceMetrics.java

```diff
@@ -0,0 +1,309 @@
+package backtraceio.library.metrics;
+
+import android.content.Context;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import backtraceio.library.common.DeviceAttributesHelper;
+import backtraceio.library.events.RequestHandler;
+import backtraceio.library.interfaces.Api;
+import backtraceio.library.interfaces.Metrics;
+import backtraceio.library.logger.BacktraceLogger;
+import backtraceio.library.models.BacktraceMetricsSettings;
+import backtraceio.library.models.json.BacktraceAttributes;
+
+public final class BacktraceMetrics implements Metrics {
+
+    private static final transient String LOG_TAG = BacktraceMetrics.class.getSimpleName();
+
+    /**
+     * Default time interval in min
+     */
+    public final static int defaultTimeIntervalInMin = 30;
+
+    /**
+     * Default time interval in milliseconds
+     */
+    public final static long defaultTimeIntervalMillis = defaultTimeIntervalInMin * 60 * 1000;
```

Comment:
wouldn't it be better if we will use `Ms` instead of `Millis`? `Ms` is more obvious but I might be wrong.